src/errors.ts

```diff
@@ -1,4 +1,4 @@
-import { z } from 'https://deno.land/x/zod@v3.21.4/mod.ts'
+import type { Schema as AnySchema, Infer } from 'npm:@decs/typeschema@0.9.1'
```

Comment:
I would rather reduce the number of dependencies. Here we are keeping the zod dependency and adding typeschema.

Reply:
I see it now.
We have asked to not change the tests :man_facepalming: